src/Summary.hs

```diff
@@ -30,7 +30,7 @@ data BuiltinValue = BuiltinValue
 
 
 dedupeBuiltin :: [(BuiltinKey, BuiltinValue)] -> [(BuiltinKey, BuiltinValue)]
-dedupeBuiltin = Map.toAscList . Map.fromListWith (curry snd)
+dedupeBuiltin = Map.toAscList . Map.fromListWith (const id)
```

Comment:
I think I'd prefer the lambda variant here. With `\_ old -> old` as that's more meaningful